### PR TITLE
Add scaling job entry point

### PR DESCRIPTION
**Pull request by oyvindeide**
This adds the entry point for the observations scaling job

### CODE REVIEW

File: share/ert/workflows/jobs/internal/scripts/enkf_scaling_job.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from res.enkf import ErtScript
+from res.enkf.jobs.scaling import scaling_job
+
+
+class EnkfScalingJob(ErtScript):
+
+    def run(self, job_config_file):
+
+        user_config = load_yaml(job_config_file)
+
```
Comment (xjules) [inline share/ert/workflows/jobs/internal/scripts/enkf_scaling_job.py]: do we need the empty lines?

File: python/tests/res/enkf/jobs/scaling/test_enkf_scaling_job.py [hunk 2]
```diff
@@ -186,6 +188,33 @@ def test_old_enkf_scaling_job(setup_ert):
...
+    obs_vector = obs["POLY_OBS"]
+
+    assert_obs_vector(obs_vector, 1.0)
+
+    job_config = {"CALCULATE_KEYS": {"keys": ["POLY_OBS"]}}
+
+    with open("job_config.yml", "w") as fout:
+        yaml.dump(job_config, fout)
+
+    job = ert.getWorkflowList().getJob("CORRELATE_OBSERVATIONS")
```
Comment (xjules) [inline python/tests/res/enkf/jobs/scaling/test_enkf_scaling_job.py:205]: More suggestions shall come:
OUTLIER_OBS_SCALING
PCA_OBS_SCALING

File: share/ert/workflows/jobs/internal/scripts/enkf_scaling_job.py (hunk 1 above)
Comment (oyvindeide) [reply share/ert/workflows/jobs/internal/scripts/enkf_scaling_job.py]: No we don't, I've removed them, you're right, easier to read that way.

File: python/tests/res/enkf/jobs/scaling/test_enkf_scaling_job.py (hunk 2 above)
Comment (oyvindeide) [reply python/tests/res/enkf/jobs/scaling/test_enkf_scaling_job.py:205]: Good suggestions, would like it to be as simple and descriptive as possible. The question is, if we expand the usage in the future, should we make a new job, or a more complex config? A more complex config where the user can for example specify type of normalization, scaling factor calculation, etc.

**Review by xjules — APPROVED** [review]
Well done!

**Pull request merged**